### ISSUE TITLE
Support for input mode

### ISSUE THREAD

**Issue by dyspop** [drac94/react-auth-code-input#22]
I would like the be able to specify the inputmode for the inputs themselves. Using this attribute displays the number pad on iOS.

```
<AuthCode
            characters={6}
            inputMode='numeric'
/>
```

**Comment by drac94** [comment]
Thanks for the suggestion @dyspop, I already created https://github.com/drac94/react-auth-code-input/pull/25 and will be released in version 2

**Comment by drac94** [comment]
Hi @dyspop I just released version 2.1.0, if you set the ìnputType` to `number` it will display the numeric keyboard on mobile. Cheers!
